testdata/sqllogictests/functions/generate_series.slt

```diff
@@ -61,3 +61,29 @@ query I
 select sum(a) from generate_series(1, 200000) as t(a);
 ----
 20000100000
+
+query R
+select * from generate_series(1.1, 4, 1.3);
+----
+1.1
+2.4000000000000005
+3.7
```

Comment:
Nice, I was expecting that a first pass might only have _just_ floats or _just_ ints for args. Cool that they can be mixed.